app/Models/LogoVote.php

```diff
@@ -42,11 +42,11 @@ class LogoVote extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->